Rakefile

```diff
@@ -45,8 +51,22 @@ namespace :app_base do
 
   task :fetch => [:clone, :ungit]
 
+  task :cleanup => :fetch do
+    require 'find'
+    migrations_dir = File.join(name, "backend/db/migrate")
+    shell.run('rm', '-rf', File.join(migrations_dir, "*"))
+    shell.run('touch', File.join(migrations_dir, '.gitkeep'))
+
+    Find.find(name) do |path|
```

Comment:
Am I missing something? This looks like it would clobber the templates every time you re-fetched.